### ISSUE TITLE
false positive no-tag rules for win_lineinfile module

### ISSUE THREAD

**Issue by davorceman** [ansible/ansible-lint#3997]
##### Summary
I have this issue and found that it is resolved with this [PR](https://github.com/ansible/ansible-lint/pull/1373)
Seems that is fixed only for lineinfile not win_lineinfile. It would be good to have it implemented there as well

##### Issue Type

- Bug Report

```console (paste below)
ansible-lint --version
ansible-lint 6.22.2 using ansible-core:2.15.8
```

**Comment by ssbarnea** [comment]
@davorceman Can you please make a PR to also address it for `win_lineinfile`? Thanks!
